docs/source/modin/supported/window_supported.rst

```diff
@@ -44,18 +44,19 @@ Rolling window functions
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``rank``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``sem``                     | N                               |                                                    |
+| ``sem``                     | P                               | ``N`` for non-integer ``window``, ``axis = 1``,    |
+|                             |                                 | or ``min_periods = 0``                             |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``skew``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``std``                     | P                               | ``N`` for non-integer ``window``, ``axis = 1``,    |
-|                             |                                 | or ``min_periods = 0``                             |
+|                             |                                 | ``min_periods = 0``, or ``ddof`` is not 0 or 1     |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``sum``                     | P                               | ``N`` for non-integer ``window``, ``axis = 1``,    |
 |                             |                                 | or ``min_periods = 0``                             |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``var``                     | P                               | ``N`` for non-integer ``window``, ``axis = 1``,    |
-|                             |                                 | or ``min_periods = 0``                             |
+|                             |                                 | ``min_periods = 0``, or ``ddof`` is not 0 or 1     |
```

Comment:
Does this still hold true (N if ``ddof`` is not 0 or 1)?

Reply:
Yes we don't support `ddof not in (0, 1)` for `var` and `std` -- see `aggregation_utils.py`